src/js/components/stock-movement-wizard/StockMovement.jsx

```diff
@@ -125,14 +127,14 @@ class StockMovements extends Component {
       newName = `${origin.name}.${destination.name}.${dateReq}.${stocklistPart}${trackingNumber}.${description}`;
       newName.replace(/ /gi, '');
     }
-    return `${values.movementNumber} - ${newName}`;
+    return values.movementNumber;
   }
 
   getAdditionalWizardTitle() {
     const { currentPage, values } = this.state;
     const shipped = values.shipped ? 'SHIPPED' : '';
     const received = values.received ? 'RECEIVED' : '';
-    if (currentPage === 6) {
+    if ((this.props.hasPackingSupport && currentPage === 6) || currentPage === 5) {
```

Comment:
Why you want to have it both on pack and send page?

Reply:
This is so the label will appear on the send page in all cases: page 6 when it has packing support and page 5 otherwise